schedule_lessons/templates/dashboard/my_profile.html

```diff
@@ -257,6 +257,9 @@ <h4 class="lesson-title">{{ lesson.name }}</h4>
             <!-- TODO: Add a image with a caption like in the empty agenda and
                        tutors/students page to show that there are no past
                      lessons -->
+
+                     <img class="no-lesson-picture" src="{% static 'dashboard/images/empty.png' %}" alt="Nolessons">
+                     <p class="text-style">Nothing here yet!</p>
```

Comment:
Could you change the font of this. Maybe one of the avenir-next's? Try them and see how they look. I feel like it would look better than plain sans-serif.

Reply:
<img width="321" alt="Screen Shot 2019-07-25 at 8 32 43 PM" src="https://user-images.githubusercontent.com/25538753/61921765-785f4c80-af1b-11e9-9539-13d09bd320ba.png">

Reply:
Good call, looks a lot better